Code/GraphMol/SmilesParse/SmilesParse.cpp

```diff
@@ -394,8 +395,13 @@ RWMol *SmilesToMol(const std::string &smiles,
       throw;
     }
     // figure out stereochemistry:
-    bool cleanIt = true, force = true, flagPossible = true;
-    MolOps::assignStereochemistry(*res, cleanIt, force, flagPossible);
+    if (params.useLegacyStereo) {
+      bool cleanIt = true, force = true, flagPossible = true;
+      MolOps::assignStereochemistry(*res, cleanIt, force, flagPossible);
```

Comment:
Do both paths run CIP?

Reply:
Nope. The new path just calls `findPotentialStereo()`, and that doesn't do CIP perception.